home/home_2.1/HelloWorld.cpp

```diff
@@ -0,0 +1,92 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <unordered_map>
+#include <sstream>
+
+const std::string FILENAME = "user_stats.txt";
+
+std::unordered_map<std::string, int> readStats() {
+    std::unordered_map<std::string, int> stats;
+    std::ifstream infile(FILENAME);
+    std::string line;
+    while (std::getline(infile, line)) {
+        std::istringstream iss(line);
+        std::string name;
+        int count;
+        if (iss >> name >> count) {
+            stats[name] = count;
+        }
+    }
+    return stats;
+}
+
+void writeStats(const std::unordered_map<std::string, int>& stats) {
+    std::ofstream outfile(FILENAME, std::ios::trunc);
+    for (const auto& pair : stats) {
+        outfile << pair.first << " " << pair.second << "\n";
```

Comment:
minor: we can use std:: format as an alternative 
outfile << std::format("{} {}\n", pair.first, pair.second);